iml-wasm-components/iml-grafana-chart/src/lib.rs

```diff
@@ -3,25 +3,38 @@
 // license that can be found in the LICENSE file.
 
 use iml_environment::grafana_root;
-use seed::{attrs, iframe, prelude::*};
+use seed::{attrs, iframe, p, prelude::*};
 
 pub static GRAFANA_DASHBOARD_ID: &str = "OBdCS5IWz";
 pub static GRAFANA_DASHBOARD_NAME: &str = "stratagem";
 
+#[derive(Debug, Default, serde::Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct GrafanaChartData<'a> {
+    pub org_id: u16,
+    #[serde(rename(serialize = "var-fs_name"))]
+    pub var_fs_name: &'a str,
+    pub refresh: &'a str,
+    pub panel_id: u16,
+}
+
 /// Create an iframe that loads the specified stratagem chart
 pub fn grafana_chart<T>(
     dashboard_id: &str,
     dashboard_name: &str,
-    fs_name: &str,
-    refresh: &str,
-    panel_id: u16,
     width: &str,
     height: &str,
+    chart_data: GrafanaChartData,
```

Comment:
This still has the same problem from earlier. This should just be a param that implements Serialize, then you won't need to care about the param values at all.

Reply:
Perhaps I misunderstood? I put all of the query parameters into the GrafanaChartData struct. The other items being passed in are not query parameters. Are those supposed to be in the struct as well?

Reply:
Changed to `impl serde::Serialize` as discussed.